python/strings_udf/strings_udf/__init__.py

```diff
@@ -1,134 +1,21 @@
 # Copyright (c) 2022-2023, NVIDIA CORPORATION.
-import glob
 import os
 
-from cubinlinker.patch import _numba_version_ok, get_logger, new_patched_linker
 from cuda import cudart
-from numba import cuda
-from numba.cuda.cudadrv.driver import Linker
 from ptxcompiler.patch import NO_DRIVER, safe_get_versions
 
+from cudf.core.udf.utils import (
+    _get_cuda_version_from_ptx_file,
+    _get_ptx_file,
+    maybe_patch_numba_linker,
```

Comment:
After this change, we end up having to decide wether or not to patch the numba linker in `cudf/__init__.py`, which is imported as part of importing `strings_udf`. As such I'm not sure we need to do so anymore here.

Reply:
No need to do so again, I think.